### PR TITLE
feat: add configuration for zoom strategy

### LINKED ISSUE #958: Add a configuration to map in builder
Add a configuration to map in builder

Image

- Add 'None' option eventually if we need to deactivate the zoom to function

### PR DESCRIPTION
**Pull request by sebbousquet**
- Fix #958

### CODE REVIEW

**Comment by sebbousquet** [comment]
Capture d’écran 2024-07-26 à 17 23 04

Capture d’écran 2024-07-26 à 17 23 02

**Review by QuCMGisaia — APPROVED** [review]

**Review by MAudelGisaia — APPROVED** [review]

**Comment by sebbousquet** [comment]
the last commit requires a toolkit release in order to merge

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -342,8 +345,8 @@
 	"Drag and drop": "Drag and drop",
 	"Drag and drop description": "Allows widgets to be moved by dragging them",
 	"Look and feel": "Look and feel",
-	"Zoom to data": "Zoom to data",
-	"Zoom to data description": "Displays a button to adjust the zoom level of the map to the available data extent",
+	"Zoom to data strategy": "Zoom to data",
+	"Zoom to data strategy description": "Choose the zoom to data strategy: \n • None: disable the feature \n • Centroid: zoom to centroid extent \n • Geometry: zoom to geometry extent",
```
Comment (MohamedHamouGisaia) [inline src/assets/i18n/en.json:349]: I suggest : 
• None: Zoom to data is disabled \n • Centroid: Zoom to the extent formed by data's centroids \n • Geometry: Zoom to the extent formed by data's geometries",

Comment (sebbousquet) [reply src/assets/i18n/en.json]: Done

**Review by QuCMGisaia — APPROVED** [review]

**Pull request merged**